Mathlib/Data/Real/ConjugateExponents.lean

```diff
@@ -59,6 +63,10 @@ theorem sub_one_pos : 0 < p - 1 := sub_pos.2 h.one_lt
 theorem sub_one_ne_zero : p - 1 ≠ 0 := ne_of_gt h.sub_one_pos
 #align real.is_conjugate_exponent.sub_one_ne_zero Real.IsConjugateExponent.sub_one_ne_zero
 
+nonrec lemma inv_pos : 0 < p⁻¹ := inv_pos.2 h.pos
+lemma inv_nonneg : 0 ≤ p⁻¹ := h.inv_pos.le
+lemma inv_ne_zero : p⁻¹ ≠ 0 := h.inv_pos.ne'
```

Comment:
These should be `protected`. Presumably the point is to use dot notation anyway, so it doesn't hurt anything.

Reply:
I wanted to keep the PR focused, but I agree, so done it is!
```suggestion
protected lemma inv_pos : 0 < p⁻¹ := inv_pos.2 h.pos
protected lemma inv_nonneg : 0 ≤ p⁻¹ := h.inv_pos.le
protected lemma inv_ne_zero : p⁻¹ ≠ 0 := h.inv_pos.ne'
```